appeals/web/src/server/appeals/appeal-details/manage-linked-appeals/manage-linked-appeals.validators.js

```diff
@@ -9,6 +9,9 @@ export const validateAddLinkedAppealReference = createValidator(
 		.notEmpty()
 		.withMessage('Enter an appeal reference')
 		.bail()
```

Comment:
This `.bail()` needs removing right?

Reply:
It's to skip the next validation step if any previous steps failed (https://express-validator.github.io/docs/api/validation-chain/#bail), so in this case it's intentional (no point running the subsequent steps if any of the previous ones failed). I think this follows the pattern used in the other validators